### PR TITLE
Doxygen updates for getgbh.f getgbm.f getgbmh.f getgbmp.f getgbp.f (Part of #77)

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
Doxygen for:
- getgbh.f
- getgbm.f
- getgbmh.f
- getgbmp.f
- getgbp.f
Depends on #92 
Depends on #91 

Part of #77

### CODE REVIEW

File: src/getgbeh.f [hunk 1]
```diff
@@ -1,167 +1,167 @@
...
-C>     processor.
+C> @note In order to unpack grib from a multiprocessing environment
+C> where each processor is attempting to read from its own pair of
+C> logical units, one must directly call subprogram getgbemh as
+C> below, allocating a private copy of cbuf, nlen and nnum to each
+C> processor. Do not engage the same logical unit from more than one
+C> processor.
 C>
-C>     @author Mark Iredell @date 94-04-01
+C> @author Mark Iredell @date 94-04-01
```
Comment (Hang-Lei-NOAA) [inline src/getgbeh.f]: date not consistent with above

File: src/getgbem.f [hunk 2]
```diff
@@ -1,208 +1,180 @@
...
-C>   SUBPROGRAM CAN BE CALLED FROM A MULTIPROCESSING ENVIRONMENT.
-C>   DO NOT ENGAGE THE SAME LOGICAL UNIT FROM MORE THAN ONE PROCESSOR.
-C>
-C> ATTRIBUTES:
-C>   LANGUAGE: FORTRAN 77
-C>   MACHINE:  CRAY, WORKSTATIONS
+C> @note Specify an index file if feasible to increase speed.
+C> Subprogram can be called from a multiprocessing environment.
+C> Do not engage the same logical unit from more than one processor.
 C>
```
Comment (Hang-Lei-NOAA) [inline src/getgbem.f]: need @ author and date

**Comment by edwardhartnett** [comment]
@Hang-Lei-NOAA you should merge these PRs when you are satisfied that they are OK, and especially that they do not change or break existing code, but only involve comments. I don't think @BrianCurtis-NOAA has admin rights on this repo, so I believe he cannot merge...

**Comment by BrianCurtis-NOAA** [comment]
@edwardhartnett I can merge it seems

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Comment by Hang-Lei-NOAA** [comment]
Okay, Thanks

**Review by Hang-Lei-NOAA — COMMENTED** [review]
These conflicts need to be address from the previous pull

**Pull request merged**